src/System.Linq.Expressions/tests/Array/NewArrayBoundsTests.cs

```diff
@@ -287,6 +287,7 @@ public static void UpdateNullThrows()
         }
 
         [Theory, ClassData(typeof(CompilationTypes))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Some exception messages stripped")]
```

Comment:
Why is this exception message stripped but the one for the MultipleNegativeBoundErrorMessage test below isn't?

And is there another test validating the OverflowException getting thrown in these situations?  I'm wondering whether the whole test should be skipped, or whether we should just conditionalize that one Assert.Equal at the end.